includes/emails/class-woothemes-sensei-email-learner-completed-course.php

```diff
@@ -61,11 +67,10 @@ function trigger( $user_id = 0, $course_id = 0 ) {
 			'passed'			=> $passed,
 		), $this->template );
 
-		// Set recipient (learner)
-		$this->recipient = stripslashes( $this->user->user_email );
-
 		// Send mail
 		$woothemes_sensei->emails->send( $this->recipient, $this->subject, $woothemes_sensei->emails->get_content( $this->template ) );
```

Comment:
With reference to this comment: https://github.com/woothemes/sensei/pull/1254/files#r51528651 . These hooks are redundant.